cli.md

```diff
@@ -33,13 +33,14 @@ The codemods must run in the given format:
 The `executable` could involve multiple command line tokens (e.g., `npm run` or `java -jar my-codemod.jar`) in order to invoke the executable.
 
 ## Implementation details
-- Passing one of `--help`, `--list`, `--describe`, and `--version` will cause the given action to be run, then exit
-- You can only run one of `--help`, `--list`, `--describe`, `--version`. Running multiple will cause an error and should show help like any other argument error
+- Passing one of `--help`, `--list`, `--describe`, and `--version` will cause the given action to be run, then exit.
+- You can only run one of `--help`, `--list`, `--describe`, `--version`. Running multiple will cause an error and should show help like any other argument error.
 - The only required field is `<project directory>`. However, this field is not required if running either `--help`, `--list`, `--describe`, or `--version`.
-- You cannot legally specify any argument more than one time
-- If `--output` is given, it indicates the path where a codetf or diff file will be created (depending on the value of `--output-format`). Otherwise no output file is generated
+- You cannot legally specify any argument more than one time.
+- If `--output` is given, it indicates the path where a codetf or diff file will be created (depending on the value of `--output-format`). Otherwise no output file is generated.
 - All codemod rules are loaded by default unless `--codemode-include` specifies a list.  `-codemode-exclude` works off all default codemods.
-- Specifying a `--codemod-include`or `--codemod-exclude`  that references a non-existent codemod will result in an error
+- Specifying a `--codemod-include` or `--codemod-exclude`  that references a non-existent codemod will result in an error
+- You can specify a simple wildcard for `--codemod-include` and `--codemod-exclude` (e.g., `--codemod-include=acme:*`). If this pattern doesn't match any codemods, an error will result.
```

Comment:
I think it should be valid to pass a pattern that doesn't match any codemods. I'm also wondering whether we should revisit the behavior for explicitly selecting a non-existent codemod. It feels to me like maybe this should be a warning instead of an error? Does this make anything easier where upstream users are concerned?

Reply:
Maybe not -- a warning sounds ok.